homeassistant/components/matter/light.py

```diff
@@ -370,20 +392,28 @@ def _update_from_device(self) -> None:
                 ):
                     supported_color_modes.add(ColorMode.COLOR_TEMP)
                     self._supports_color_temperature = True
-                    min_mireds = self.get_matter_attribute_value(
-                        clusters.ColorControl.Attributes.ColorTempPhysicalMinMireds
+                    min_mireds = (
+                        self.get_matter_attribute_value(
+                            clusters.ColorControl.Attributes.ColorTempPhysicalMinMireds
+                        )
+                        or DEFAULT_MIREDS_MIN
                     )
-                    if min_mireds > 0:
-                        self._attr_max_color_temp_kelvin = (
-                            color_util.color_temperature_mired_to_kelvin(min_mireds)
+                    self._attr_max_color_temp_kelvin = (
+                        color_util.color_temperature_mired_to_kelvin(
+                            max(min_mireds, DEFAULT_MIREDS_MIN)
                         )
-                    max_mireds = self.get_matter_attribute_value(
-                        clusters.ColorControl.Attributes.ColorTempPhysicalMaxMireds
                     )
-                    if max_mireds > 0:
-                        self._attr_min_color_temp_kelvin = (
-                            color_util.color_temperature_mired_to_kelvin(max_mireds)
+                    max_mireds = (
+                        self.get_matter_attribute_value(
+                            clusters.ColorControl.Attributes.ColorTempPhysicalMaxMireds
                         )
+                        or DEFAULT_MIREDS_MAX
+                    )
+                    self._attr_min_color_temp_kelvin = (
+                        color_util.color_temperature_mired_to_kelvin(
+                            min(max_mireds, DEFAULT_MIREDS_MAX)
+                        )
+                    )
```

Comment:
One more small thing: Previously, we checked that the device value needs to be bigger than `0`.
If `max_mireds` were to be negative (e.g. `-1`), we'd now provide that to `color_temperature_mired_to_kelvin`, giving us `-1000000` kelvin for `_attr_min_color_temp_kelvin`.

But I guess matter-wise, it's not possible to actually get any negative values here due to the data type, so fine to remove this check?

Reply:
@TheJulianJES min/max mitred are uint on matter side so if there is ever coming a negative value then more stuff is broken